composer.json

```diff
@@ -58,6 +59,10 @@
             "install-frameworkmissing.html",
             "install.php",
             "web.config"
+        ],
+        "expose": [
```

Comment:
Because we were serving the site from the root before, we didn't need to expose anything.